tests/package-search-test.js

```diff
@@ -148,7 +148,7 @@ describeApplication('PackageSearch', () => {
       });
 
       it('displays an empty search', () => {
-        expect(PackageSearchPage.$searchField).to.have.value('');
+        expect(PackageSearchPage.$titleSearchField).to.have.value('');
```

Comment:
Since titles are resources in our app `titleSearchField` feels a little confusing. Maybe we should go with `nameSearchField` since all of our resources have `name` attributes

Reply:
Only thing here is that we are referring to the title search element from the package search. It seems like name would apply to package and provider in addition to title. We could call them titleSearchField, packageSearchField and providerSearchField distinctly in all pages provider-search.js, title-search.js and package-search.js

Reply:
Ah, I see. I think what confused me is in the gif we're calling the title's name field `title` so I had assumed `$titleSearchField` was meant to be the name search field.